docs/layouts/shortcodes/swoole.md

```diff
@@ -0,0 +1,17 @@
+## Install
+
+Install the PHP extension for Swoole or Open Swoole during the build.
+
+Take advantage of an [installation script](https://raw.githubusercontent.com/platformsh/snippets/main/src/install_swoole.sh).
+You need to pass 2 parameters:
+
+ - Which Swoole project to use: `openswoole` or `swoole`
+ - Which version to install
+
+```yaml {location=".platform.app.yaml"}
+hooks:
+    build: |
+        set -e
+        ...
+        sh <( curl -s 'https://raw.githubusercontent.com/platformsh/snippets/main/src/install_swoole.sh') openswoole 4.11.1
```

Comment:
Unfortunately, the `<` in the code block currently gets escaped ([shows up as `&lt;`](https://pr-2385-qud5sma-652soceglkw4u.eu-3.platformsh.site/guides/laravel/deploy/octane.html#install). See how to get around this issue: https://github.com/platformsh/platformsh-docs/blob/main/contributing/markup-format.md#reuse-content

Reply:
I tried to move the .md to a .html file. The content appears similar but the code is kind of crappy.

I tried also to include a shortcode in a shortcode but I couldn't. It was to have a reusable build hook executing stuff from `platformsh/snippets`.

Reply:
You can't do shortcodes inside shortcodes (an unfortunate limitation to Hugo), but you can use other methods. I created a PR to show one way it can be done: https://github.com/thomasdiluccio/platformsh-docs/pull/1 It definitely looks nicer that way. I think I'll probably update our style guide and workaround to use that way.

Reply:
Yet @CollierCZ, I don't manage to build the doc. The readFile part doesn't seem to be recognized triggering `hugo serve`

Reply:
Well ... doc is building without including the install script ... So no.

Reply:
Sorry, @thomasdiluccio , that was my fault with the syntax. Hope it should work now. Seems to locally.